plugins/test/dynamic_test.cc

```diff
@@ -194,11 +239,32 @@ void DynamicTest::TestBody() {
     CheckPhaseResults("response_headers", invoke.result(), stream, res);
   }
   if (cfg_.response_body_size() > 0) {
-    for (auto& invoke : *cfg_.mutable_response_body()) {
-      auto res = stream.SendResponseBody(std::move(
-          *absl::WrapUnique(invoke.mutable_input()->release_content())));
-      ASSERT_VM_HEALTH("response_body", handle, stream);
-      CheckPhaseResults("response_body", invoke.result(), stream, res);
+    if (cfg_.chunking_plan_case() !=
```

Comment:
This is very similar to the earlier logic from lines 205--230, can you factor it out into a helper method that can be called from both places?